lib/prawn/table/cell/text.rb

```diff
@@ -135,7 +135,13 @@ def text_box(extra_options={})
         #
         def styled_width_of(text)
           options = @text_options.reject { |k| k == :style }
-          with_font { @pdf.width_of(text, options) }
+          if text.empty?
```

Comment:
I personally would've preferred `.max || 0` but it's not critical.

Is this case covered by a spec?